.cirrus.yml

```diff
@@ -1,3 +1,36 @@
+# Global defaults
+timeout_in: 120m  # https://cirrus-ci.org/faq/#instance-timed-out
+container:
+  # https://cirrus-ci.org/faq/#are-there-any-limits
+  # Each project has 16 CPU in total, assign 2 to each container, so that 8 tasks run in parallel
+  cpu: 2
+  memory: 6G  # https://cirrus-ci.org/guide/linux/#linux-containers
+env:
+  PACKAGE_MANAGER_INSTALL : "apt-get update && apt-get install -y"
+  MAKEJOBS: "-j4"
```

Comment:
This setting does not corresponds with `cpu: 2`, no?

Reply:
Everything should be identical to what it is on travis (except the timeout)

This is the default value, see

```
ci/test/00_setup_env.sh:export MAKEJOBS=${MAKEJOBS:--j4}